linux_os/guide/system/bootloader-grub2/grub2_mds_argument/rule.yml

```diff
@@ -0,0 +1,56 @@
+documentation_complete: true
+
+prodtype: fedora,rhel9
+
+title: 'Configure Microarchitectural Data Sampling mitigation'
+
+description: |-
+    Microarchitectural Data Sampling (MDS) is a hardware vulnerability which allows unprivileged
+    speculative access to data which is available in various CPU internal buffers.
+
+    When performing store, load, L1 refill operations, processors write data into temporary
+    microarchitectural structures (buffers), and the data in the buffer can be forwarded to load
+    operations as an optimization.
+
+    Under certain conditions, data unrelated to the load memory address can be speculatively
```

Comment:
I don't understand "load memory address" here. Is it target address of the "load" operation?

Reply:
Yes.

Reply:
```suggestion
    Under certain conditions, data unrelated to the load operations can be speculatively
```